app/services/events/EventsService.scala

```diff
@@ -36,13 +37,22 @@ trait EventsService {
 
   def eventFileParsingService: EventsParsingService
 
-  def saveAssessmentEvents(): Future[Unit] = {
+  def eventsConfigRepo: EventsConfigRepository
+
+  def saveAssessmentEventsFromDeprecatedCsv(): Future[Unit] = {
     eventFileParsingService.processCentres().flatMap { events =>
       Logger.debug("Events have been processed!")
       eventsRepo.save(events)
     }
   }
 
+  def saveAssessmentEvents(): Future[Unit] = {
+    eventsConfigRepo.events.flatMap { events =>
```

Comment:
I know, it's not part of spike, but probably if you have time, it would be good to make more robust solution.
There is possibility to save event with invalid location/venue/event type (yes, you wrap it in object, but it never check if such value is in config). 
For example there is nice logic of creating venue from name based on config in `EventsController`: 
`locationsAndVenuesRepository.venue(venueParam).flatMap { venue =>`
The same validation can be done for most enum-like fields.